charts/rancher-monitoring/v0.2.2/questions.yml

```diff
@@ -0,0 +1 @@
+rancher_min_version: 2.5.6-rc1
```

Comment:
@aiyengar2 should we max out 0.2.1 to 2.5.7 ?

Reply:
We need `charts/rancher-monitoring/v0.2.1/questions.yml to have `rancher_max_version: 2.5.7` and this questions.yml to have `rancher_min_version: 2.5.8-rc1` right?

Reply:
Given the change, I think we should max old one, but do we need min on this one ?